init.d/modules.BSD.in

```diff
@@ -0,0 +1,24 @@
+#!@SBINDIR@/openrc-run
+
+description="Loads a user defined list of kernel modules."
+
+depend() {
+	after bootmisc
+	keyword -jail -prefix
+}
+
+start()
+{
+	local cnt=0 x
+	for x in $modules
+	do
+		ebegin "Loading module $x"
+		kldload "$x";
+		eend $? "Failed to load $x" && : $(( cnt += 1 ))
```

Comment:
Are you referring to removing both lines 12, 17, and replacing 17 with "Autoloaded $x module(s)"?  FreeBSD uses kldload.  I did not quite understand how Linux was loading the module from the Linux example.